surfsense_backend/app/services/document_processing/docling_service.py

```
@@ -0,0 +1,346 @@
+            # Set SSL environment for EasyOCR downloads
+            os.environ['CURL_CA_BUNDLE'] = ''
+            os.environ['REQUESTS_CA_BUNDLE'] = ''
+            
+            # Try to use local models first, fallback to download if needed
+            try:
+                reader = easyocr.Reader(['en'], 
+                                       download_enabled=False,
+                                       model_storage_directory="/root/.EasyOCR/model")
+                logger.info("✅ EasyOCR configured for local models")
+                return reader
+            except:
```

Ruff (0.12.2)

Do not use bare `except`

(E722)